translations/pagerfanta.eu.xlf

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en" target-language="eu" datatype="plaintext" original="none">
+        <body>
+            <trans-unit id="previous">
+                <source>previous</source>
+                <target>aurrekoa</target>
```

Comment:
did you try the result in kimai? these are the translations in the pager below each datable and for my personal preference: I don't like if there is a translation inside (thats why they are empty in all languages):
<img width="1005" alt="Bildschirmfoto 2020-01-17 um 12 56 37" src="https://user-images.githubusercontent.com/533162/72610859-ce400a80-3928-11ea-99a1-6af63f6b8d37.png">